src/sentry/options/defaults.py

```diff
@@ -689,6 +689,7 @@
 # project config computation. This is temporary option to monitor the performance of this feature.
 register("dynamic-sampling:boost-latest-release", default=False)
 register("dynamic-sampling.prioritise_projects.sample_rate", default=0.0)
+register("dynamic-sampling.sliding_window_rebalancing.sample_rate", default=0.0)
```

Comment:
As I said before, not sure we need this ( I think we don't).
The query you are doing is already tested for performance and it has no issues.